extension/sqlsmith/statement_generator.cpp

```diff
@@ -109,6 +150,76 @@ unique_ptr<CreateStatement> StatementGenerator::GenerateCreate() {
 	return create;
 }
 
+unique_ptr<AttachStatement> StatementGenerator::GenerateAttach() {
+	auto attach = make_uniq<AttachStatement>();
+	attach->info = GenerateAttachInfo();
+	return attach;
+}
+
+unique_ptr<DetachStatement> StatementGenerator::GenerateDetach() {
+	auto detach = make_uniq<DetachStatement>();
+	detach->info = GenerateDetachInfo();
+	return detach;
+}
+
+// generate USE statement
+unique_ptr<SetStatement> StatementGenerator::GenerateSet() {
+	auto name_expr = make_uniq<ConstantExpression>(Value("RANDOM_STRING_" + RandomString(10)));
```

Comment:
Instead of `"RANDOM_STRING" + RandomString(10);`, can you write something like 

```
string StatementGenerator::GenerateTableIdentifier() {
	auto identifier = "t" + to_string(GetIndex());
	current_relation_names.push_back(identifier);
	return identifier;
}
```
but for database names?